### DISCUSSION TITLE
RuntimeError: Task 0 is not successful!

### DISCUSSION THREAD

**Discussion by warriors1523** [deepmodeling/dpgen#700] [Q&A]
Hello developers,
When I run the instruction “dpgen autotest run relaxation. json machine.json", the following error occurs:
![image](https://user-images.githubusercontent.com/92622311/163545458-7a3bb927-505e-4264-9513-449799f4df88.png)

The machine configuration file is as follows
![image](https://user-images.githubusercontent.com/92622311/163545581-8be0fcc3-61dd-4b61-8e43-dad0c5ff88f4.png)

What task do this command "run_tasks[ii]" in the file "common_equi.py "
![image](https://user-images.githubusercontent.com/92622311/163668596-8d927b1e-7b72-4ab6-8648-24d9504e83e9.png)

Besides, how can I fix this bug
![image](https://user-images.githubusercontent.com/92622311/163580407-e233e14b-6b8f-4bb5-8ca0-84f5411cd714.png)

p.s. The machine configuration file can run successfully during training the model.And I can manually run the task successfully with the command "lmp-in in.lammps"

Looking forward to your reply.
zhang xj

**Comment by wanghan-iapcm** [comment]
Could you please go to the `work_path` to check if the submission script is correct in the sense that you can manually submit the jobs and the job finishes successfully.

**Reply by warriors1523** [reply]
When  I manually submit the jobs(relaxation task),  the following error occurs:
![image](https://user-images.githubusercontent.com/92622311/163952600-e2c12b94-0667-4c9e-948a-bf70ce3827fc.png)

After adding "* *" to pair coeff, I manually submit the relaxation task, and the job finishes successfully. But the command "dpgen autotest run relaxation.json machine.json" is still failed and the mistake is the same as before. So I wonder if there is a problem with the machine.json. However, machine.json can run successfully in the stage of "dpgen run". 
![图片](https://user-images.githubusercontent.com/92622311/163954771-daf12535-d636-4e32-b359-bf28b14b12f7.png)

DPGEN:0.10.3     deepmd-kit:2.1.0
This makes me very confused.
Looking forward to your reply.
zhang xj
